frontend/src/lib/components/PropertyFilters/components/PropertyFilterButton.tsx

```diff
@@ -1,37 +1,24 @@
 import './PropertyFilterButton.scss'
 import { Button } from 'antd'
-import { useValues } from 'kea'
-import { formatPropertyLabel, midEllipsis } from 'lib/utils'
 import React from 'react'
-import { cohortsModel } from '~/models/cohortsModel'
 import { AnyPropertyFilter } from '~/types'
-import { keyMapping } from 'lib/components/PropertyKeyInfo'
-import { propertyDefinitionsModel } from '~/models/propertyDefinitionsModel'
 import { CloseButton } from 'lib/components/CloseButton'
 import { IconCohort, IconPerson, UnverifiedEvent } from 'lib/components/icons'
 import { Tooltip } from 'lib/components/Tooltip'
+import { cohortsModel } from '~/models/cohortsModel'
+import { useValues } from 'kea'
+import { propertyDefinitionsModel } from '~/models/propertyDefinitionsModel'
+import { formatPropertyLabel, midEllipsis } from 'lib/utils'
+import { keyMapping } from 'lib/components/PropertyKeyInfo'
 
 export interface PropertyFilterButtonProps {
     onClick?: () => void
     onClose?: () => void
-    children?: string | JSX.Element
+    children?: string
     item: AnyPropertyFilter
     style?: React.CSSProperties
 }
 
-export function PropertyFilterText({ item }: PropertyFilterButtonProps): JSX.Element {
```

Comment:
As part of this, I refactored this out NOT to be a component, as it was only used in two places, and was overly complex (wrapping a string into a React fragment).

Reply:
No reason to block the improvement on some strokey-beard philosophising

Not being a component means the two places it was used can be changed independently without some realising that's what they're doing. Is that desirable?

Reply:
I thought about that, but these are fairly simple function calls and used in exactly two places. My DRY sense usually starts tingling on three repeats. ;)

Reply:
Damp not DRY 👍

Reply:
Erm.... (D)on't (A)dd (M)ethods (P)rematurely 🤣